### DISCUSSION TITLE
Отключение точки доступа

### DISCUSSION THREAD

**Discussion by Oldroid123** [vvip-68/LedPanelWiFi#61] [Вопросы и ответы]
Приветствую. Возможно, есть у кого-то идеи, что и где подправить в прошивке, чтобы точка доступа отключалась условно через 5 минут, если к ней никто не подключен? WiFi сети для подключения рядом так же нет

**Comment by NanoProto** [comment]
Привет, используйте таймер timerMinim и проверку на подключение, что то типа client.connected
